index.html

```diff
@@ -3,9 +3,67 @@
   <head>
     <meta charset="UTF-8" />
     <title>Tribute Page</title>
-    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="css/style.css" />
```

Comment:
The style.css file is not linked correctly to the HTML file.

Reply:
- [ ] As a way to construct our code review in a professional way 👍, let's always include an explanation, a "why", for each requested change. 

- [ ] Please kindly include a reason for your suggested changes 👏 as the reasons might not be clear to the code author if we didn't explain them. 🤷‍♂️

Reply:
The style.css file is not linked correctly to the HTML file 🤕, and the styles that you applied didn´t work. You forgot that the styles.css file is inside a folder, please correct the route and take it into account next time. 👍